src/com/company/CommandLineImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.company;
+
+import java.util.Scanner;
+import java.util.UUID;
+
+public class CommandLineImpl implements CommandLine {
```

Comment:
czy tu na pewno jest wrzucony master akutalny do tego brancha? bo te zmiany powinny miec na masterze to chyba nie jest up to master